CHANGES.rst

```diff
@@ -816,6 +822,7 @@ astropy.io.votable
 
 astropy.modeling
 ^^^^^^^^^^^^^^^^
+- Propagate weights to underlying fitter in ``FittingWithOutlierRemoval`` [#7020]
```

Comment:
Need to add `#7249` as well to the log.

Reply:
please just have the number of this PR rather than as well, as the other PR is obsolete/duplicate once this one is merged.